[2025-03-17T22:30:25.610+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T18:37:53.643064+00:00 [queued]>
[2025-03-17T22:30:25.683+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T18:37:53.643064+00:00 [queued]>
[2025-03-17T22:30:25.684+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T22:30:27.983+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_data_phase_1_task> on 2025-03-16 18:37:53.643064+00:00
[2025-03-17T22:30:28.078+0000] {standard_task_runner.py:60} INFO - Started process 15991 to run task
[2025-03-17T22:30:28.098+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_data_phase_1_task', 'manual__2025-03-16T18:37:53.643064+00:00', '--job-id', '193', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmpx067poga']
[2025-03-17T22:30:28.113+0000] {standard_task_runner.py:88} INFO - Job 193: Subtask transform_data_phase_1_task
[2025-03-17T22:30:30.300+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T18:37:53.643064+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T22:30:33.630+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_data_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T18:37:53.643064+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T18:37:53.643064+00:00'
[2025-03-17T22:30:33.637+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T22:30:33.638+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T22:30:33.723+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T22:30:55.140+0000] {subprocess.py:93} INFO - [0m22:30:55  Running with dbt=1.9.0-b2
[2025-03-17T22:31:00.529+0000] {subprocess.py:93} INFO - [0m22:31:00  Registered adapter: postgres=1.8.2
[2025-03-17T22:31:03.905+0000] {subprocess.py:93} INFO - [0m22:31:03  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T22:31:03.905+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T22:31:03.906+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T22:31:07.936+0000] {subprocess.py:93} INFO - [0m22:31:07  Found 6 models, 1 source, 424 macros
[2025-03-17T22:31:07.943+0000] {subprocess.py:93} INFO - [0m22:31:07
[2025-03-17T22:31:07.960+0000] {subprocess.py:93} INFO - [0m22:31:07  Concurrency: 4 threads (target='dev')
[2025-03-17T22:31:07.970+0000] {subprocess.py:93} INFO - [0m22:31:07
[2025-03-17T22:31:13.571+0000] {subprocess.py:93} INFO - [0m22:31:13  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T22:31:17.564+0000] {subprocess.py:93} INFO - [0m22:31:17  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 7[0m in 3.96s]
[2025-03-17T22:31:17.898+0000] {subprocess.py:93} INFO - [0m22:31:17
[2025-03-17T22:31:17.936+0000] {subprocess.py:93} INFO - [0m22:31:17  Finished running 1 table model in 0 hours 0 minutes and 9.92 seconds (9.92s).
[2025-03-17T22:31:18.360+0000] {subprocess.py:93} INFO - [0m22:31:18
[2025-03-17T22:31:18.361+0000] {subprocess.py:93} INFO - [0m22:31:18  [32mCompleted successfully[0m
[2025-03-17T22:31:18.361+0000] {subprocess.py:93} INFO - [0m22:31:18
[2025-03-17T22:31:18.362+0000] {subprocess.py:93} INFO - [0m22:31:18  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T22:31:22.664+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T22:31:24.190+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_data_phase_1_task, execution_date=20250316T183753, start_date=20250317T223025, end_date=20250317T223124
[2025-03-17T22:31:25.275+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T22:31:26.133+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
